test/integration/serverless/snapshots/trace-java

```diff
@@ -90,8 +89,6 @@
               "cold_start": "true",
               "dd_extension_version": "123",
               "env": "integration-tests-env",
-              "function.request": "{}",
-              "function.response": "{}",
```

Comment:
I feel like we should still keep these tags, even when there's nothing in the request/response. That information could be helpful to users. So they could see the response was `{}`.

Reply:
This is actually a bug. Good catch. My brain somehow ignored these changes.